EIPS/eip-9999.md

```diff
@@ -0,0 +1,296 @@
+---
+eip: 9999
+title: Enshrined Proposer-Builder Separation
+description: Separates the ethereum block in consensus and execution parts, adds a mechanism for the consensus proposer to choose the execution proposer.
+author: Francesco D'Amato <francesco.damato@ethereum.org>, Barnabé Monnot <barnabe.monnot@ethereum.org>, Michael Neuder <michael.neuder@ethereum.org>, Potuz (@potuz), Terence Tsao <ttsao@offchainlabs.com>
+discussions-to: https://ethereum-magicians.org/t/epbs-discusion-topic/19634
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7732-enshrined-proposer-builder-separation-epbs/19634
```
I changed the title on Eth Magicians to add the EIP number and the full title of the EIP, but feel free to change.